test/fcmv1_test.exs

```diff
@@ -268,6 +266,64 @@ defmodule Sparrow.FCM.V1Test do
     assert config.authentication == auth
   end
 
+  test "FCM accounts are passed correctly" do
```

Comment:
add negative test to check how it behaves in error case

Reply:
What exactly is the error case here? Missing file? Invalid file? In both cases application will not start

Reply:
all of the following ;)